src/Auditable.php

```diff
@@ -103,13 +103,15 @@ protected function resolveAuditExclusions()
             }
         }
 
-        // Valid attributes are all those that made it out of the exclusion array
-        $attributes = array_except($this->attributes, $this->excludedAttributes);
-
-        foreach ($attributes as $attribute => $value) {
-            // Apart from null, non scalar values will be excluded
-            if (is_array($value) || (is_object($value) && !method_exists($value, '__toString'))) {
-                $this->excludedAttributes[] = $attribute;
+        if (!Config::get('audit.nonScalar', false)) {
+            // Valid attributes are all those that made it out of the exclusion array
+            $attributes = array_except($this->attributes, $this->excludedAttributes);
+
+            foreach ($attributes as $attribute => $value) {
+                // Apart from null, non scalar values will be excluded
+                if (\is_array($value) || (\is_object($value) && !method_exists($value, '__toString'))) {
```

Comment:
Also, remove the backslash from `is_array()` and `is_object()`.

Reply:
Sure...